src/components/SideBySide.js

```diff
@@ -9,13 +9,29 @@ import { isMdxType } from '../utils/mdx';
 const SideBySide = ({ className, children, type }) => {
   const types = Array.isArray(type) ? type : [type];
   const childObjects = Children.toArray(children);
+
+  const isGatsbyMdxType = (child, type) => {
```

Comment:
This name is a bit misleading and doesn't really tell me how it differentiates with `isMdxType`. Is there something we can do to make this more obvious at what this function does?

[nit] Since this function does not use any of its outer scope, you can pull it out of the component.

Reply:
Hmm yeah, I named it this way because it's still supposed to ultimately be checking if the type we're passing in matches the type of the child we're looking at. But since we know the images will always be nested we have to add this extra check. Happy to rename it, but having a hard time! any suggestions?